api/soap/mantisconnect.wsdl

```diff
@@ -288,6 +332,20 @@
    <xsd:element name="require_closed" type="xsd:boolean" minOccurs="0"/>
   </xsd:all>
  </xsd:complexType>
+ <xsd:complexType name="FilterCustomField">
+  <xsd:all>
+   <xsd:element name="name" type="xsd:string" minOccurs="0"/>
+   <xsd:element name="id" type="xsd:integer" minOccurs="0"/>
+   <xsd:element name="value" type="tns:StringArray" minOccurs="0"/>
```

Comment:
Should we require the value for custom field?  Otherwise, why would the record exist?